tests/unit/Logger.js

```diff
@@ -113,7 +113,7 @@ describe('WereLogs Logger is usable:', () => {
         const logger = new Logger('test');
         assert.doesNotThrow(
             () => {
-                logger.setLevel('fatal');
+                logger.setLevel('error');
```

Comment:
I think it makes more sense to add this is as another test. This way we are testing if it's working as expected when logLevel === dumpLevel and if logLevel < dumpLevel.